app/retail/emails.py

```diff
@@ -565,36 +565,36 @@ def render_new_bounty_roundup(to_email):
 '''
     highlights = [
         {
-            'who': 'hardlydifficult',
+            'who': 'eiselems',
             'who_link': True,
-            'what': 'Worked with Unlock Protocol on their smart contract.',
-            'link': 'https://gitcoin.co/issue/unlock-protocol/unlock/172/991',
+            'what': 'Completed the first Status Embark bounty in 48 hours!',
+            'link': 'https://gitcoin.co/issue/embark-framework/embark/765/1089',
             'link_copy': 'View more',
         },
         {
-            'who': 'KevinLiLu',
+            'who': 'lastmjs',
             'who_link': True,
-            'what': 'Worked with CyberCongress on their Ethereum uncle rewards!',
-            'link': 'https://gitcoin.co/issue/cybercongress/cyber-search/185/1043',
+            'what': 'Worked with Piper and Carver at the Ethereum Foundation on web3py.',
+            'link': 'https://gitcoin.co/issue/ethereum/web3.py/962/1036',
             'link_copy': 'View more',
         },
         {
-            'who': 'StevenJNPearce',
+            'who': 'bakaoh',
             'who_link': True,
-            'what': 'Helped MARKET add a method to find a deployed contract address.',
-            'link': 'https://gitcoin.co/issue/MARKETProtocol/MARKET.js/145/989',
+            'what': 'Back at it, this time on the OmiseGO Plasma MVP repo.',
+            'link': 'https://gitcoin.co/issue/omisego/plasma-mvp/170/967',
             'link_copy': 'View more',
         },
     ]
 
     bounties_spec = [
         {
-            'url': 'https://github.com/ethereum/ethereum-org/issues/898',
-            'primer': 'Have Design chops? The Ethereum Foundation is looking to design a grants website. Apply this weekend!',
```

Comment:
Have Design chops sounded cool :P

Reply:
@thelostone-mc We'll bring it back next newsletter 😄